delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -202,21 +195,18 @@ def _prepare_attributes(
             "przl": services,
             "weight": total_weight,
         }
-        option_codes = [
-            packaging.postlogistics_option_code
-            for packaging in postlogistics_packagings
-        ]
+        option_code = packaging.postlogistics_option_code
```

Comment:
With your previous change, service codes (DLC) are now recorded in the field shipper_package_code, so we don't need anymore the field postlogistics_option_code. Many services (=options) can be activated.

Reply:
fixed in https://github.com/OCA/delivery-carrier/pull/323/commits/f04b053b0aeca3c51756f5ad9129349ce26f4555